etc/eth-contracts/hardhat.config.js

```diff
@@ -1,17 +1,24 @@
 require('@nomiclabs/hardhat-waffle');
 require('solidity-coverage');
+require('./tasks/storage');
 
 /**
  * @type import('hardhat/config').HardhatUserConfig
  */
 module.exports = {
+    newStorageLayoutPath: './storageLayout',
```

Comment:
Where this variable is used later?

Reply:
This will be used to export the storage layout in a md file.